doc/install/containers.rst

```diff
@@ -3,6 +3,20 @@
 Ceph Container Images
 =====================
 
+.. important::
+
+   Using the ``:latest`` tag is discouraged. If you use the ``:latest`` 
+   tag, there is no guarantee that the same image will be on each of 
+   your hosts.  Under these conditions, upgrades might not work 
+   properly. Remember that ``: latest`` is a relative tag, and a moving
+   target.
+
+   Instead of the ``: latest`` tag, use explicit tags or image ids. For
```

Comment:
```suggestion
   Instead of the ``: latest`` tag, use explicit tags or image IDs. For

Reply:
Depends on your style guide, but capitalised is always best.

Reply:
I've added this suggestion:
s/ids/IDs/